frontend/yarn.lock

```diff
@@ -1172,9 +1172,9 @@
     regenerator-runtime "^0.13.4"
 
 "@babel/runtime@^7.3.1", "@babel/runtime@^7.4.4", "@babel/runtime@^7.8.3":
-  version "7.16.3"
```

Comment:
Hey we decided to not commit yarn.lock files for now. Could you remove this from your PR? Just do `git checkout origin/development yarn.lock`, then commit and push